docs/data/base/guides/overriding-component-structure/OverridingInternalSlot.tsx

```diff
@@ -4,7 +4,17 @@ import { Option } from '@mui/base/Option';
 
 export default function OverridingInternalSlot() {
   return (
-    <Select slots={{ listbox: 'ol' }} defaultValue="First option">
+    <Select
+      slots={{ listbox: 'ol' }}
+      slotProps={{
+        listbox: {
+          style: {
+            backgroundColor: 'white',
```

Comment:
The issue with the white background color is that, in the dark theme, it's impossible to see the text.

<img width="231" alt="Screenshot 2024-02-05 at 14 15 41" src="https://github.com/mui/material-ui/assets/37222944/4786c5de-0099-495b-b714-b3b3daf5224b">

Reply:
Opps! Missed it, Updated in this [commit](https://github.com/mui/material-ui/pull/40952/commits/6184c3a188f2b29cdcac806bacf41f536103df9a)